cmd/bigsky/main.go

```diff
@@ -135,6 +138,21 @@ func run(args []string) {
 			otel.SetTracerProvider(tp)
 		}
 
+		if cctx.Bool("datadog") {
```

Comment:
Should at least have this be specified in the flag list, I don't think this args should be hidden. Also putting it in the flags lets an `EnvVars` be configured, which is how we will probably configure this up when deployed.

Do we not need to specify a sidecar endpoint or anything? Is that handled automatically by the datadog library? Would be good to document any such env vars in `./cmd/bigsky/README.md`, even if just for Jake and myself.